style.css

```diff
@@ -4,8 +4,7 @@
 }
```

Comment:
### Horizontal Scrollbar and White Spacing on Right Side of Mobile Version
![Screenshot from 2022-08-31 14-48-56 (copy)](https://user-images.githubusercontent.com/93039359/187651356-1d82f29a-a900-4ec1-8078-8bcbd89de593.png)

- [ ] You have a white spacing on right side of your mobile version. Please remove it to make your project look professional and also it is creating a horizontal scroll :sob: Please remove the white spacing to remove the horizontal scrollbar to make your project look professional :thinking: